media-libs/libsdl2/files/libsdl2-2.24.0-fix-build-without-joystick.patch

```diff
@@ -0,0 +1,31 @@
+From 71fb91f7e43c5f046a037bf5ca59214d93fe2d51 Mon Sep 17 00:00:00 2001
```

Comment:
Please pop a link to the commit at the top of this file.

Reply:
Should be fixed now